cmd/keymasterd/app.go

```diff
@@ -481,22 +481,23 @@ func (state *RuntimeState) writeHTML2FAAuthPage(w http.ResponseWriter,
 	return nil
 }
 
-func (state *RuntimeState) writeHTMLLoginPage(w http.ResponseWriter, r *http.Request,
-	loginDestination string, errorMessage string) error {
-	//footerText := state.getFooterText()
+func (state *RuntimeState) writeHTMLLoginPage(w http.ResponseWriter,
```

Comment:
why change the signature to remove the err?

Reply:
We're not using the error anywhere, so this code has been dead for a long time (possibly stillborn).